index.html

```diff
@@ -34,7 +33,7 @@
             </nav>
             
         </header>
-        </div>
+        
```

Comment:
- [ ] Please work on your desktop design view following the Figma [Design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1)
### **Take a look at your design**
![head](https://user-images.githubusercontent.com/55596970/167155462-cc195588-ff10-4e43-8926-80250245afa6.png)
### **This is Figmal Design**
![destop](https://user-images.githubusercontent.com/55596970/167155617-82446b1d-c0fc-4a21-b154-6080709db94d.png)